packages/transaction-controller/src/types.ts

```diff
@@ -78,6 +78,11 @@ type TransactionMetaBase = {
    */
   origin?: string;
 
+  /**
+   * The transaction's 'r' value as a hex string.
```

Comment:
Maybe `The transaction signature 'r' value as a hex string.` and the same format for the others?